spec.md

```diff
@@ -53,6 +53,7 @@ These headers are OPTIONAL and clients SHOULD NOT depend on them.
 Several terms are used frequently in this document and warrant basic definitions:
 
 - **Registry**: a service that handles the required APIs defined in this specification
+- **Repository**: a namespace within the registry, commonly represented here as `<name>` between the `/v2/` and API call
```

Comment:
repository is a bit more than just a namespace..

hmm..

Reply:
how's that?

Reply:
@mikebrow if we strip out the "in the registry" text from other parts, would a more minimal definition of Repository that doesn't get into isolation and authentication make sense? I'm trying to mirror the other definitions in their brevity.

Reply:
even after removing the "in the registry" text we still have a number of uses of the term repository throughout the spec that hint around at the normal uses of the term. The last sentence could be moved to the requirements section to make it clear that although the specification mentions repositories, it does not restrict or define their namespace isolation models authentication apis etc..

Reply:
note the  "definition" part was fairly thin.. "a concept in registries for isolating API requests from one connection to another"

Reply:
```suggestion
- **Repository**: a concept in registries for isolating API requests from one connection to another -- such as for isolating public and private access to the registry in the execution of the APIs in this specification. Referred herein as a namespace in the registry and as`<name>` in the API examples. **Note**: This specification does not limit the registry implementation of repositories to any type of namespace or namespace model, nor does this specification define how the registry performs isolation/authentication of the connections.
```

Reply:
Circling back to this, the reasoning for using the "namespace" term is because we defined it with the following text throughout the spec:

> `<name>` is the namespace of the repository

I'm avoiding mentioning access restrictions because the spec doesn't go there anywhere else and we don't have a spec for authentication and authorization defined yet (though I would like to see that added). I think the most we can say is that references with a descriptor (index to manifest, manifest to blob) are within a repository and don't cross that boundary. The best word I can think of for that is "namespace" since we've used it so many other places in the spec already.

> a concept in registries for isolating API requests from one connection to another

I'm not a fan of defining it based on connections. It implies that the pull of a manifest in one connection could reference blobs in another repository by having a separate connection for that blob. We've leaned pretty heavily on saying that all descriptors are references within the same repository and registries depend on that for implementing access and GC on top of this spec.